src/Symfony/Component/ClassLoader/ClassCollectionLoader.php

```diff
@@ -106,6 +106,7 @@ public static function load($classes, $cacheDir, $name, $autoReload, $adaptive =
 
         $c = '(?:\s*+(?:(?:#|//)[^\n]*+\n|/\*(?:(?<!\*/).)++)?+)*+';
         $strictTypesRegex = str_replace('.', $c, "'^<\?php\s.declare.\(.strict_types.=.1.\).;'is");
+        $haltCompilerRegex = str_replace('.', $c, "'.__halt_compiler.\(.\).'is");
```

Comment:
Right. The trailing dot for sure, the leading one need at least a '\b'. I'll fix it.